Set .Sx in italics, not in roman with quotes (local regression fix).

Index: tmac/mdoc/doc-ditroff
--- tmac/mdoc/doc-ditroff.orig
+++ tmac/mdoc/doc-ditroff
@@ -73,6 +73,7 @@
 .ds doc-Pa-font \f[I]
 .ds doc-Sh-font \f[\*[HF]]\"
 .ds doc-Sy-font \f[B]
+.ds doc-Sx-font \f[I]
 .ds doc-Tn-font \f[R]
 .ds doc-Va-font \f[I]
 .ds doc-Xr-font \f[I]
